file-formats/apob/apob-v1.json

```diff
@@ -0,0 +1,128 @@
+{
+  "$comment": "This file is autogenerated, do not edit manually, see https://github.com/SAP/abap-file-formats for more information.",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/apob/apob-v1.json",
+  "title": "Application Object",
+  "description": "Type of application object",
+  "type": "object",
+  "properties": {
+    "formatVersion": {
+      "title": "Format Version",
+      "description": "Format version",
+      "type": "string",
+      "const": "1"
+    },
+    "header": {
+      "title": "Header",
+      "description": "Header",
+      "type": "object",
+      "properties": {
+        "description": {
+          "title": "Description",
+          "description": "Description of the ABAP object",
+          "type": "string",
+          "maxLength": 60
+        },
+        "originalLanguage": {
+          "title": "Original Language",
+          "description": "Original language of the ABAP object",
+          "type": "string",
+          "minLength": 2
+        },
+        "abapLanguageVersion": {
+          "title": "ABAP Language Version",
+          "description": "ABAP language version",
+          "type": "string",
+          "enum": [
+            "standard",
+            "keyUser",
+            "cloudDevelopment"
+          ],
+          "enumTitles": [
+            "Standard",
+            "ABAP for Key Users",
+            "ABAP Cloud Development"
+          ],
+          "enumDescriptions": [
+            "Standard",
+            "ABAP for key user extensibility",
+            "ABAP cloud development"
+          ],
+          "default": "standard"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "description",
+        "originalLanguage"
+      ]
+    },
+    "attributes": {
+      "title": "Attributes",
+      "description": "Attributes of the application object",
+      "type": "object",
+      "properties": {
+        "aoKeyStructure": {
+          "title": "Key Table / Structure",
+          "description": "Name of key table or structure for application object",
+          "type": "string",
+          "maxLength": 30
+        },
+        "sapObjectType": {
```

Comment:
```suggestion
        "sapObject": {
```

Reply:
Object's name which we are storing in our table, is "SAP Object Type". We cannot adept this suggestion.